lib/nxt_state_machine/state_machine.rb

```diff
@@ -75,6 +75,10 @@ def any_state
       states.values.map(&:enum)
     end
 
+    def any_state_except(*excluded)
+      any_state - excluded
+    end
+
```

Comment:
I feel you might be able to use `alias_method` here also

```suggestion
    alias_method :all_states_except, :any_state_except
```